File: trunk/quests/freportw/Hyrill_Pon.pl

############################################
# ZONE: West Freeport (freportw)
# DATABASE: PEQ-Velios
# LAST EDIT DATE: April 5,2005
# VERSION: 2.0
# DEVELOPER: MWMDRAGON
#
# *** NPC INFORMATION ***
#
# NAME: Hyrill_Pon
# ID: 9060
# TYPE: Rogue
# RACE: Human
# LEVEL: 8
#
# *** ITEMS GIVEN OR TAKEN ***
#
# Torn Parchment ID-18010
#
# *** QUESTS INVOLVED IN ***
#
#1 - Hollow Skull
#
# *** QUESTS AVAILABLE TO ***
#
#1 - Enchanter , Magician , Necromancer
#
############################################

sub EVENT_SAY
{
 if($text=~/Hail/i)
 {
   quest::say("What do you want?  Leave me be!  Can't a man drown his sorrows in peace!?");
 }
}

sub EVENT_ITEM
{
 if($class != 'Enchanter' || $class != 'Magician' || $class != 'Necromancer' || $class != 'Wizard')
 {
 quest::say("I am sorry. But you have no business giving ne this junk.");
 quest::summonitem($item1) if($item1);
 quest::summonitem($item2) if($item2);
 quest::summonitem($item3) if($item3);
 quest::summonitem($item4) if($item4);
 return;
 }
 else
     {
 # Torn Parchment ID-18010
 if($itemcount{18010} == 1)
 {
   quest::say("Peh! He thinks this old skull he found is a legendary skull of Wun Toque. It is said, a wizard who possesses one is granted power and intelligence far beyond those of his peers. Yiz was searching for the skulls missing ruby eyes. It seems he found the location of the first eye. Hmm.. Lynuga.. Lynuga.. I think I have heard that name before.. Yeah! Now I remember. I met her in the Foreign Quarter of Neria.. um.. Highpass Hold. She was trying to hawk some stolen gems! I think she mumbled something about going home to Grobb. I sure don't have time to track her down.");
   quest::ding();
   quest::exp(1000);
    # Commons Residents Faction
    quest::faction("51","3");
    # Freeport Militia Faction
    quest::faction("105","3");
    # Knights of Truth Faction
    quest::faction("184","3");
    # Priests of Marr Faction
    quest::faction("258","3");
 }
     }

}

#END of FILE Zone:freportw  ID:9060 -- Hyrill_Pon.pl

File: freeportwest/Hyrill_Pon.lua
function event_say(e)
	if(e.message:findi("hail")) then
		e.self:Say("What do you want? Leave me be! Can't a man drown his sorrows in peace!?");
	end
end

function event_trade(e)
	local item_lib = require("items");

	if(e.other:Class() == "Enchanter" or e.other:Class() == "Magician" or e.other:Class() == "Necromancer" or e.other:Class() == "Wizard") then
		if(item_lib.check_turn_in(e.trade, {item1 = 18010})) then -- Torn Parchment
			e.self:Say("Peh! He thinks this old skull he found is a legendary skull of Wun Toque. It is said, a wizard who possesses one is granted power and intelligence far beyond those of his peers. Yiz was searching for the skulls missing ruby eyes. It seems he found the location of the first eye. Hmm.. Lynuga.. Lynuga.. I think I have heard that name before.. Yeah! Now I remember. I met her in the Foreign Quarter of Neria.. um.. Highpass Hold. She was trying to hawk some stolen gems! I think she mumbled something about going home to Grobb. I sure don't have time to track her down.");
			e.other:Ding();
			e.other:Faction(51,3,0);
			e.other:Faction(105,3,0);
			e.other:Faction(184,3,0);
			e.other:Faction(258,3,0);
			e.other:Faction(0,0,0);
			e.other:AddEXP(1000);
		end
	end
	item_lib.return_items(e.self, e.other, e.trade)
end

-- END of FILE Zone:freportw -- Hyrill_Pon.pl